### DISCUSSION TITLE
How to adding service account on vps?

### DISCUSSION THREAD

**Discussion by anti-government** [arghyac35/aria-telegram-mirror-bot#182] [General]
so i have allready generate service account, but i dont know how to run the bot with service account in vps. can someone tell me how to do it ? accounts folder allready in dir

**Comment by hsj51** [comment]
> so i have allready generate service account, but i dont know how to run the bot with service account in vps. can someone tell me how to do it ? accounts folder allready in dir

In .constants.js, set `USE_SERVICE_ACCOUNT` and `IS_TEAM_DRIVE` to `true` and keep all the service accounts in root of accounts folder.
You also need to add all the service accounts to the teams/shared drive as content manager.
